samtranslator/swagger/swagger.py

```diff
@@ -47,19 +47,12 @@ def __init__(self, doc):
         modifications on this copy.
 
         :param dict doc: Swagger document as a dictionary
-        :raises ValueError: If the input Swagger document does not meet the basic Swagger requirements.
         """
 
-        if not SwaggerEditor.is_valid(doc):
-            raise ValueError("Invalid Swagger document")
-
         self._doc = copy.deepcopy(doc)
-        self.paths = self._doc["paths"]
-        self.security_definitions = self._doc.get("securityDefinitions", Py27Dict())
-        self.gateway_responses = self._doc.get(self._X_APIGW_GATEWAY_RESPONSES, Py27Dict())
-        self.resource_policy = self._doc.get(self._X_APIGW_POLICY, Py27Dict())
-        self.definitions = self._doc.get("definitions", Py27Dict())
+        self.validate_definition_body(doc)
 
+        self.paths = self._doc.get("paths")
```

Comment:
Out of curiosity, was was `_doc["paths"]` changed to `_doc.get("paths")`?

Reply:
I think at some point I had the `self.validate_definition_body(doc)` line after `self.paths = self._doc["paths"]`. So if `paths` wasn't there, it would throw a runtime error (vs. with `.get` it returns `None`). However, shouldn't make a difference now that validation happens before.